app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
Ok it seems to be non-deterministic, because now I don't get the error after removing it from the Jail.

I wonder if changing the constant doesn't always happen reliably.

https://github.com/theforeman/foreman/blob/develop/app/services/foreman/plugin.rb#L227